catalog/resource_sql_table.go

```diff
@@ -189,7 +189,7 @@ func (ti *SqlTableInfo) serializeColumnInfo(col SqlColumnInfo) string {
 	if col.Comment != "" {
 		comment = fmt.Sprintf(" COMMENT '%s'", parseComment(col.Comment))
 	}
-	return fmt.Sprintf("%s %s%s%s", col.Name, col.Type, notNull, comment) // id INT NOT NULL COMMENT 'something'
+	return fmt.Sprintf("%s %s%s%s", getWrappedColumnName(col), col.Type, notNull, comment) // id INT NOT NULL COMMENT 'something'
```

Comment:
FYI: probably a more consistent way to do this is either 1) make a constructor for `ColumnInfo` that escapes the column name, or 2) define a method on `ColumnInfo` that returns the escaped name.